packages/web-react/src/components/Checkbox/demo/CheckboxHiddenLabel.tsx

```diff
@@ -2,7 +2,13 @@ import React from 'react';
 import Checkbox from '../Checkbox';
 
 const CheckboxHiddenLabel = () => (
-  <Checkbox id="checkbox-hidden-label" name="checkboxHiddenLabel" label="Checkbox Label" isLabelHidden />
+  <Checkbox
+    id="checkbox-hidden-label"
+    name="checkboxHiddenLabel"
+    label="Checkbox Label"
+    isLabelHidden
+    onChange={() => {}}
```

Comment:
**question (if-minor):** Why is there a repeating `onChange` prop with an empty function? Must it be there?

Reply:
During the development, I watched the console, and there were errors about the checkbox not having any onChange method when it is selected. So, I added onChange methods everywhere to resolve these errors and clear up the console.